server/zanata-frontend/src/frontend/app/styles/style.less

```diff
@@ -6035,6 +6035,14 @@ i.i--left.i--lock, i.i--left.i--trash {
   padding-bottom:@spacing-base-and-a-half;
 }
 
+.translationContainer pre.cm-s-default {
+  word-break: break-word;
```

Comment:
I think this should be break-all or normal (break-word appears to be a word-wrap value?)